packages/graph/hash_graph/lib/graph/src/types/mod.rs

```diff
@@ -1,50 +1,74 @@
-//! Descriptions, and implementation logic, of the Types of elements of the graph.
+//! Model types used across datastores
 
-use sqlx::{types::Uuid, FromRow};
+use std::fmt;
 
-#[derive(Debug, FromRow)]
-pub struct DataType {
-    pub version_id: Uuid,
-    pub schema: serde_json::Value,
-    pub created_by: Uuid,
+use uuid::Uuid;
+
+#[repr(transparent)]
+#[derive(Clone, Copy, Debug, sqlx::Type, PartialEq, Eq)]
+#[sqlx(transparent)]
```

Comment:
***Tiny*** nit: putting the `transparent` attributes next to each other looks better. Also I'd prefer the derived traits to list the std-ones first, then third party ones. Personally, I always use this order: `Debug`, `Copy`, `Clone`, comparisons (`PartialEq`, `Eq`, `PartialOrd`, `Ord`), `Hash`, but that's a matter of taste. Feel free to ignore these suggestions.
```suggestion
#[repr(transparent)]
#[sqlx(transparent)]
#[derive(Debug, Copy, Clone, PartialEq, Eq, sqlx::Type)]
```

Reply:
`sqlx::Type` has to come before `sqlx(transparent)` I believe.

Reply:
Hmm, that makes sense but annoys me 😄